app/models/distribution_template.rb

```diff
@@ -0,0 +1,14 @@
+# encoding: utf-8
+
+class DistributionTemplate < BaseModel
+  belongs_to :distribution
+  belongs_to :audio_version_template
+
+  def account
+    distribution.try(:account)
```

Comment:
Yeah, that makes sense.  Dist-owner is responsible for the relation, rather than the AudioVersion owner.  (Though they're almost certainly the same).

Reply:
probably need a validation to make sure they both belong to the same series as well